assets/src/containers/FilesAccessed.js

```diff
@@ -160,9 +179,27 @@ function FilesAccessed (props) {
     }
   }
 
+  const onChangeFileHandler = event => {
+    const value = event.target.value
+    if (event.target.checked == true && !resourceFilter.includes(value)) {
```

Comment:
`event.target.checked == true` is redundant, it should just be `event.target.checked`